apps/web/src/components/AdPanel/Variations/ContentAd.tsx

```diff
@@ -0,0 +1,17 @@
+import { BodyText } from '../BodyText'
+import { AdButton } from '../Button'
+import { AdCard } from '../Card'
+import { AdPlayerProps } from '../types'
+import { getImageUrl } from '../utils'
+
+export const ContentAd = (props: AdPlayerProps) => {
+  return (
+    <AdCard imageUrl={getImageUrl('ad_clamm_options_trading.png')} {...props}>
+      <BodyText fontSize="15px">Web3 Notifications Now Live!!</BodyText>
```

Comment:
Should add translation here?

Reply:
Components in `Variations` are for reference. Have added translations 👍🏻